src/frontend/apps/desk/src/app/InnerLayout.tsx

```diff
@@ -0,0 +1,48 @@
+import { Loader } from '@openfun/cunningham-react';
+import { useEffect } from 'react';
+
+import useAuthStore from '@/auth/useAuthStore';
+import { Box } from '@/components';
+import { HEADER_HEIGHT, Header } from '@/features/header';
+import { Menu } from '@/features/menu';
```

Comment:
suggestion: should we have only one export for all the '@/features/...'?

Reply:
https://github.com/numerique-gouv/people/blob/c05aef8f57e17a3e8a1b8674628299fb6f312427/src/frontend/apps/desk/src/features/index.ts